index.html

```diff
@@ -5,25 +5,27 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <title>Portfolio</title>
+    <title>Goddy's Portfolio</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
   <header class="navigation">
-    <a href="#" class="logo">My Logo</a>
+    <a href="#" class="logo"><./Goddy></a>
     <nav>
-      <ul class="desktop-header">
+      <ul class="desktop-header" id="dh">
         
-        <li>
+        <li id="portfolio">
           <a href="#portfolio">Portfolio</a>
         </li>
-        <li>
-          <a href="#about">About</a>
+        <li id="about">
+          <a href="#About">About</a>
         </li>
-        <li>
-          <a href="#contact">Contact</a>
+        <li id="contact">
+          <a href="#Contact">Contact</a>
         </li>
+        <li class="close" id="close">&times;</li>
       </ul>
-      <ul class="nav">
+      <ul class="nav" id="hamb">
```

Comment:
- [ ] Good job so far implementing the mobile responsive menu, :+1: :100: But there are some improvements you need to make before you move to the next milestone. 
-  It seems like your mobile menu implementation is also taking effect on desktop, Clicking on a nav item when on desktop actually opens an overlay on top of the entire screen, please you need to fix this as it's against the design and also not optimal for the user experience. :smile: :wave: 
- SCREENSHOT FOR REFERENCE:
![Screenshot from 2023-04-18 00-15-18](https://user-images.githubusercontent.com/107632780/232631567-c5268dbd-35a3-4dca-8354-49b7f627840e.png)